### PR TITLE
new attr, tests improv

### PR DESCRIPTION
**Pull request by Loloxon**

### CODE REVIEW

File: graph/elements/node.py [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
+    def __init__(self, x: float, y: float, is_hanging: bool, is_production_relevant: bool = True, label: str = "V"):
         """
         Initializes a Node object.
 
         Args:
             x (float): The x-coordinate of the node in a 2D space.
             y (float): The y-coordinate of the node in a 2D space.
             is_hanging (bool): A boolean attribute representing "hanging node" a property (true means node is hanging).
-            label (str, optional): A label for the node. Defaults to "V".
+            label (str, optional): A label for the node. Defaults to .
```
Comment (strjakub) [inline graph/elements/node.py:10]: "Defaults to ."

File: productions/p1.py [hunk 2]
```diff
@@ -12,10 +12,10 @@ def _name(self) -> str:
 
     @property
     def _left_side(self) -> HyperGraph:
-        nodes = [Node(0, 0, False, "V"),
-                 Node(1, 0, False, "V"),
-                 Node(1, 1, False, "V"),
-                 Node(0, 1, False, "V")]
+        nodes = [Node(0, 0, False, False),
```
Comment (strjakub) [inline productions/p1.py:15]: it is production relevant

File: graph/elements/node.py (hunk 1 above)
Comment (Loloxon) [inline graph/elements/node.py:10]:
```suggestion
            label (str, optional): A label for the node. Defaults to "V".
```

File: productions/p1.py (hunk 2 above)
Comment (Loloxon) [inline productions/p1.py:15]:
```suggestion
        nodes = [Node(0, 0, False),
```

**Review by strjakub — APPROVED** [review]

**Pull request closed without merging**